reference/SimpleCalc/MVVM-CSharp/SimpleCalculator/MainViewModel.cs

```diff
@@ -1,24 +1,14 @@
-﻿using CommunityToolkit.Mvvm.ComponentModel;
-using CommunityToolkit.Mvvm.Input;
-
 namespace SimpleCalculator;
 
 public partial class MainViewModel : ObservableObject
 {
-	private readonly IThemeService _themeService;
+    private readonly IThemeService _themeService;
```

Comment:
Can you make sure that spacing/tabs is consistent with rest of the code

Reply:
@nickrandolph used tabs. But the `editorconfig` used by the template is set to space and not tabs. I found that's the default on our template.
![image](https://github.com/unoplatform/Uno.Samples/assets/20712372/2cea9c23-988e-4295-a9c7-6017e77feb47)

Reply:
happy for the code to change to tabs but we should apply that across all code, not just on one file